### PR TITLE
Add basic style to displayed messages

### PR DESCRIPTION
**Pull request by valevassallo**
Things done:

- Changed renderComments function
- Added user avatar/icon
- Added style to the messages

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -185,12 +185,20 @@ function renderComments(channelName) {
...
     const divMessage = document.createElement("div");
 
-    divMessage.innerHTML = `<p class="message-item">${
-      channelMessage.author
-    } ${savedTime.toLocaleTimeString(undefined, {
-      hour: "2-digit",
-      minute: "2-digit"
-    })}</p> <p>${channelMessage.content}</p>`;
+    divMessage.innerHTML = `<div class="single-message">
+      <img src="./assets/user_icon.png" alt="user's icon">
```
Comment (MariethPerez) [inline public/js/main.js]: Owww It is very interesting that you use an icon.

Comment (AngieCristina) [reply public/js/main.js]: Nice work, add an icon similar to the original version

Comment (CesarCachay) [reply public/js/main.js]: Great! the attribute "alt" has an special purpose or it's like the name of the icon?

Comment (valevassallo) [reply public/js/main.js]: @CesarCachay it's something like a description

**Pull request merged**